OIPA/iati_synchroniser/models.py

```diff
@@ -2,22 +2,24 @@
 import datetime
 from django.utils.safestring import mark_safe
 from django.utils.translation import ugettext_lazy as _
-from iati_synchroniser.dataset_syncer import DatasetSyncer
 from iati.parser import Parser
 from iati.deleter import Deleter
 
-INTERVAL_CHOICES = (
-    (u'YEARLY', _(u"Parse yearly")),
-    (u'MONTHLY', _(u"Parse monthly")),
-    (u'WEEKLY', _(u"Parse weekly")),
-    (u'DAILY', _(u"Parse daily")),
-)
 
 class Publisher(models.Model):
     org_id = models.CharField(max_length=100)
-    org_abbreviate = models.CharField(max_length=55)
+    org_abbreviate = models.CharField(max_length=55, default="")
     org_name = models.CharField(max_length=255)
-    default_interval = models.CharField(verbose_name=_(u"Interval"), max_length=55, choices=INTERVAL_CHOICES, default=u'MONTHLY')
+    default_interval = models.CharField(
+        verbose_name=_(u"Interval"),
+        max_length=55,
+        choices=(
+            (u'YEARLY', _(u"Parse yearly")),
```

Comment:
No clue, looked it up but found no reason why this could be so I removed it. Seems to work fine.